packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-in/ios/integrationsteps.md

```diff
@@ -1,43 +1,66 @@
-## Sample code
+## Integration steps
+
+### Summary
+
+There are many different ways to integrate the Swift SDK into your app. Using
+the sample code as a wrapper around the SDK, this guide describes the [steps](#steps)
+involved in integrating this use case into your application. Feel free to modify
+the sample code to fit your individual needs. The diagram below illustrates
+how the sample code and SDK fit into the overall integration.
+
+<div class="common-image-format">
 
-The following sample code illustrates how to implement basic sign-in with username and password.
+ ![Diagram showing the integration flow of the sample app and Swift SDK](/img/oie-embedded-sdk/oie-embedded-sdk-swift-sample-code-overview.png
+ "Swift sample and sdk diagram")
 
-> **Note:** The Swift SDK and sample app are dynamic in nature and do not follow a
-prescriptive and explicit flow used in these examples. These examples are meant to be a
-learning tool to help you understand how to implement specific use cases using the SDK.
-Although, you can implement such examples in your own native application, you won't
-be able to reap the benefits of dynamic authentication such as a policy driven logic
-that require no changes when policy configurations are changed.
+</div>
 
-### Executing code
+### Steps
 
-The following example shows how to execute the basic login code described in
-[Implementing code](#implementing-code). Note that the completion handler supplied
-to the `login` function will be invoked once, either with a fatal error, or with
-a token.
+#### Step 1: Initialize BasicLogin
+
+The first step is to initialize `BasicLogin` and pass in a
+`configuration` object.
 
  ```swift
- self.authHandler = BasicLogin(configuration: configuration)
- self.authHandler?.login(username: "user@example.com",
-                         password: "secretPassword")
- { result in
-     switch result {
-     case .success(let token):
-         print(token)
-     case .failure(let error):
-         print(error)
-     }
- }
+self.authHandler = BasicLogin(configuration: configuration)
 ```
 
-### Implementing code
+For more information on how to set the `configuration` parameter, see
+[SDK](/docs/guides/oie-embedded-common-download-setup-app/ios/main/#sdk)
+in
+[Download and set up the SDK, Sign-In Widget, and sample app](/docs/guides/oie-embedded-common-download-setup-app/ios/main)
+
 
-The following code implements the basic login functionality.
+#### Step 2: Call the login method
+
+The next step is to call the `login` method passing in the username
```

Comment:
In that case, since Swift/Objective-C use the attribute names as part of the method signature, it would be `login(username:password:completion:)` which is a bit verbose.  So in this case, `login` might be sufficient.

Reply:
I've changed it login() for now. We can discuss further and iterate over the convention. My preference is going with the Swift convention.

Reply:
If that's a standard preference in Swift/Objective-C, then I'm happy to go with that.

Reply:
After discussions with @mikenachbaur-okta, we agreed it's cleaner to use the method name without the parenthesis and arguments.

Reply:
OK